x-pack/plugins/alerting/server/lib/wrap_scoped_cluster_client.ts

```diff
@@ -134,6 +135,7 @@ function getWrappedTransportRequestFn(opts: WrapEsClientOpts) {
           } - ${JSON.stringify(params)} - with options ${JSON.stringify(requestOptions)}`
         );
         const result = (await originalRequestFn.call(opts.esClient.transport, params, {
+          requestTimeout: opts.requestTimeout,
```

Comment:
should we reverse this? otherwise if a rule executor specifies a request timeout in its options, it'll override the configured one.

Reply:
Can we add the requestTimeout value to the debug logger above?

Reply:
Since `requestTimeout` can be undefined, should we only pass it in if defined?

Reply:
Resolved in this commit [a71807b](https://github.com/elastic/kibana/pull/175484/commits/a71807bcd872ae278c19718fe48d8d64864a6e53)